components/CrosswordLayout/index.js

```diff
@@ -25,51 +24,81 @@ const StyledOl = styled.ol`
   grid-row: 1;
 `;
 
-export default function CrosswordLayout() {
+export default function CrosswordLayout({ onCurrentClueId, crosswordClues }) {
   return (
     <>
       <StyledTable>
         <tbody>
           <Tr>
             <Td isBlack></Td>
-            <Td></Td>
+            <Td>
+              <sup>1</sup>
+              {crosswordClues[0].isCorrectlyAnswered ? "M" : null}
```

Comment:
If the false condition only returns `undefined` you can write this as 
`{crosswordClues[0].isCorrectlyAnswered && "M"}`
That way you can save yourself writing the `: null` statement.

Got this nice tip from Anja 💪